content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -47,10 +47,10 @@ The limit applies to the entire cluster, so it affects all Nodes.
 
 ## Dynamic volume limits
 
-{{< feature-state state="beta" for_k8s_version="v1.12" >}}
+{{< feature-state state="stable" for_k8s_version="v1.17" >}}
 
-Kubernetes 1.11 introduced support for dynamic volume limits based on Node type as an Alpha feature.
-In Kubernetes 1.12 this feature is graduating to Beta and will be enabled by default.
+Support for dynamic volume limits was introduced as an Alpha feature in Kubernetes 1.11.
```

Comment:
Now that the feature is stable, it's (IMO) OK to omit the history. Readers can look at the v1.16 docs (one before the version it went GA) if they are interested.

Reply:
Re-worded, PTAL